conda-forge.yml

```
@@ -8,4 +8,7 @@ conda_forge_output_validation: true
 github:
   branch_name: main
   tooling_branch_name: main
+provider:
+  linux_aarch64: default
+  linux_ppc64le: default
```

I had a similar crash on `mysql-connector-c` and it worked fine on Azure, i.e.,

```suggestion
  linux_ppc64le: azure
```

could do the trick.